### ISSUE TITLE
ExecuteTableValidations on documents with no tables in some pages

### ISSUE THREAD

**Issue by lz01** [aws-samples/amazon-textract-response-parser#87]
Is it expected behaviour that the function `ExecuteTableValidations` stops looping through pages if it finds a page that does not contain any table?

By modifying the `break` statement to a `continue` statement here:
```
if len(current_page.tables) == 0:
            page_compare_proc += 1
            break
```
we can make sure to loop through other pages to check for mergeable tables.

**Comment by douglasqian** [comment]
Fixed here:
https://github.com/aws-samples/amazon-textract-response-parser/pull/120

**Comment by athewsey** [comment]
Closing as fixed by merge of #120 in February and no recent feedback - Please do feel free to re-open if still facing the issue on the current version of TRP!
